APMrover2/Rover.h

```diff
@@ -553,6 +556,28 @@ class Rover : public AP_HAL::HAL::Callbacks {
     bool disarm_motors(void);
     bool is_boat() const;
 
+    enum Failsafe_Action {
+        Failsafe_Action_None          = 0,
+        Failsafe_Action_RTL           = 1,
+        Failsafe_Action_Hold          = 2,
+        Failsafe_Action_SmartRTL      = 3,
+        Failsafe_Action_SmartRTL_Hold = 4,
+        Failsafe_Action_Terminate     = 5
```

Comment:
This actually matches the order in `APMrover2/failsafe.cpp` for the g.fs_action parameter. The next logical step is to make the normal failsafe handler use this same enum (and actually collapse the handling further). (This future intent is actually why these are called Failsafe_Action instead of Battery_Failsafe)